File: Periode 3/Chapter 1/2 Arrays/Arrays/UserStory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Cryptography.X509Certificates;
using System.Text;
using System.Threading.Tasks;

namespace Arrays
{
    class UserStory
    {
        private int Hours;
        private string Description;

        public UserStory(int hours, string description)
        {
            this.Hours = hours;
            this.Description = description;
        }

        public int RetHours()
        {
            return Hours;
        }

        public string RetDescription()
        {
            return this.Description.ToString();
        }
    }
}

File: Periode 3/Java/Chapter 1/3. Constructors and Collections/Exercise9/src/UserStory.java
/**
 * Created by Joost on 23-2-2017.
 */
public class UserStory
{
    public int Hours;
    public String Description;


    public UserStory(int hours, String description)
    {
        this.Hours = hours;
        this.Description = description;
    }

    public String ToString()
    {
        String output = "";
        output += this.Hours;
        return output;
    }

}
